webapp/src/main/webapp/WEB-INF/messages.properties

```diff
@@ -506,6 +506,8 @@ Concept.find=Find Concept(s)
 Concept.retiredMessage=This concept is retired
 Concept.name=Concept Name
 Concept.name.atLeastOneRequired=At least one non-empty name is required
+Concept.description.atLeastOneRequired=At least one non-empty Description is required
+Concept.description.existing.required=Existing Description can't be Empty
```

Comment:
I wouldn't complain about an empty description, the concept save handler should just remove such descriptions, i updated the ticket description to say this